### ISSUE TITLE
HD Output with Live Demo

### ISSUE THREAD

**Issue by mvoodarla** [Rudrabha/Wav2Lip#585]
Hey folks - really love Wav2Lip, but just like everyone else, I wish it had commercial usage rights and produced higher quality output.

Just thought I'd share this if it's interesting. It's the best quality we've been able to achieve (though it's not Wav2Lip). It's also much faster than other approaches like the open source [video-retalking](https://github.com/OpenTalker/video-retalking) repo but still maintains a great level of quality. I've attached a link to a demo you can try + sample output below!

https://www.sievedata.com/functions/sieve/video_retalking

https://github.com/Rudrabha/Wav2Lip/assets/11367688/f064e1f2-90e1-4ce4-9ec3-9b2c4bcbd4ea

P.S. I work at the company that built this demo. Hope we can open-source all the code very very soon!

**Comment by wallezen** [comment]
that's cool! do you have any plans to open source it now?

**Comment by mvoodarla** [comment]
yes! we've found a couple more improvements we are still in the process of making. but after we do that + clean up the code we will.
